reproductions/AutomapperTest/AutomapperTest.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFrameworks>netcoreapp2.2;netcoreapp2.1;netcoreapp2.0</TargetFrameworks>
+    <TargetFrameworks>netcoreapp3.0;netcoreapp2.2;netcoreapp2.1;netcoreapp2.0</TargetFrameworks>
```

Comment:
We should start dropping `netcoreapp2.0` everywhere soon. Maybe now is a good time? Support ended one year ago today.

Reply:
Good call. I did a clean up of the reproductions projects in 60d0681caa1f6a10aaccab537806c714ead22add.